packages/flutter_tools/test/commands.shard/hermetic/run_test.dart

```diff
@@ -1693,3 +1723,41 @@ class FakeAnsiTerminal extends Fake implements AnsiTerminal {
   @override
   bool get singleCharMode => setSingleCharModeHistory.last;
 }
+
+/// A Fake that implements FeatureFlags and enables web.
+class FakeFeatureFlags extends Fake implements FeatureFlags {
+  @override
+  bool get isWebEnabled => true;
```

Comment:
This feature flag should most likely be removed from everywhere (cc @kevmoo?). Why do we need this fake?